templates/packages/order_edit.html

```diff
@@ -36,7 +34,14 @@ <h6 class="text-danger">*קיימים שינויים לא שמורים</h6>
             <div class="col-xxl-3 col-xl-4 col-lg-4 col-md-6 col-sm-6 mb-3">
                 <div class="card h-100">
                     <div class="card-body text-start">
-                        <h5 class="card-title">{{package.destination}}</h5>
+                        <div class="row">
+                            <div class="col-auto text-decoration-none">
+                                <a type="button" data-bs-toggle="popover" data-bs-placement="bottom" data-bs-content="הערות: {{package.notes|default:'<אין>'}} מחיר: {{package.rate}} ₪"><h5 class="bi-three-dots"></h5></a>

```

Comment:
The idea is nice, but it doesn't seem to accommodate long comments well. It looks... cramped

![popover](https://i.imgur.com/q4wICgD.png)

Reply:
It's not actually critical to look well, it won't be looked at much anyway. Do you have a better idea for it?

Reply:
You could put the price on the card and not in the _popover_

Reply:
If you want to make the changes, go ahead. If not, I've approved the PR.

Reply:
Yeah I'll put the price on the card itself

Reply:
Man idk where to put it

Reply:
@tomtzook  What do you think?

Reply:
> 
> 
> @tomtzook What do you think?

About where I placed it

Reply:
Yeah, looks good.